noxfile.py

```diff
@@ -0,0 +1,75 @@
+import os
+import pathlib
+import shutil
+
+import nox
+
+nox.options.reuse_existing_virtualenvs = True
+
+
+# Use this for venv envs nox -s test
+@nox.session(python=["3.8", "3.9", "3.10", "3.11"])
+def test(session):
+    session.install(".[all]")
+    session.install("-r", "requirements-test.txt")
+    session.run(
+        "pytest", "stravalib/tests/unit/", "stravalib/tests/integration/"
+    )
+
+
+# Use this for conda/mamba = nox -s test_mamba
+@nox.session(venv_backend="mamba", python=["3.8", "3.9", "3.10", "3.11"])
+def test_mamba(session):
+    session.install(".[all]")
+    session.install("-r", "requirements-test.txt")
```

Comment:
```suggestion
    session.install("-r", "requirements.txt")
```

Reply:
@enadeau just so you can see - i was playing with various requirements files and definitely made a mistake in not updating these functions!! each one of these functions defines a session.

you can create whatever type of session you want. for instance the docs session allows us to build the docs and they will update in realtime as we work on them! If you just run nox it will run all sessions - the way i have things here that wouldn't be ideal as there is a conda and a venv tests option.